src/scarabee/_scarabee/python/cmfd.cpp

```diff
@@ -0,0 +1,183 @@
+          "damping", &CMFD::damping,
+          &CMFD::set_damping,
+          "set CMFD damping factor for under-relaxing the nonlinear diffusion coefficient"
+          "between iterations.")
+    
+  .def_property(
+          "skip_moc_iterations", &CMFD::skip_moc_iterations,
+          &CMFD::set_skip_moc_iterations,
+          "Set # of MOC iterations to skip before using CMFD")
+
+  .def_property(
+          "unbounded_cmfd_solves", &CMFD::num_unbounded_solves,
+          &CMFD::set_num_unbounded_solves,
+          "Set # of CMFD solves before flux update ratios are clamped")
+
+  .def_property(
+          "check_neutorn_balance", &CMFD::neutron_balance_check,
+          &CMFD::set_neutron_balance_check,
+          "Check neutron balance in each CMFD tile on each CMFD solve")
+
+  .def("get_surface", &CMFD::get_surface,
+      "Assigns the CMFD surface info to a segement end if it exists\n\n"
+      "Parameters\n"
+      "----------\n"
+      "point: list of float\n"
+      "       list of x and y position of segment end\n"
+      "direction: list of float\n"
+      "           list of x and y components of segment direction unit vector\n"
+      "Returns\n"
+      "-------\n"
+      "surface: CMFDSurfaceCrossing object\n"
+      "         Contains the surface crossing information for the segement end\n",
+      py::arg("point"),py::arg("direction"))
+      
+  .def("get_tile", &CMFD::get_tile,
+      "Finds the tile that a segement end exists in\n\n"
+      "Parameters\n"
+      "----------\n"
+      "point: list of float\n"
+      "       list of x and y position of segment end\n"
+      "direction: list of float\n"
+      "           list of x and y components of segment direction unit vector\n"
+      "Returns\n"
+      "-------\n"
+      "surface: CMFDSurfaceCrossing object\n"
+      "         Contains the surface crossing information for the segement end\n",
+      py::arg("point"),py::arg("direction"))
+  .def("tally_current",&CMFD::tally_current,
+      "Tallies the MOC current onto the appropriate CMFD surface(s)\n\n"
+      "Parameters\n"
+      "----------\n"
+      "aflx: float\n"
+      "      value of the angular flux to be tallied"
+      "u: Direction\n"
+          "Direction of the flux\n"
+      "G: unsigned int\n"
+      "   Int which gives the # of the flux energy group"
+      "surf: CMFDSurfaceCrossing"
+      "      CMFDSurfaceCrossing object which gives the information about the surface to tally on\n"
+      "Returns\n"
+      "-------\n"
+      "None",
+      py::arg("aflx"),py::arg("u"),py::arg("G"),py::arg("surf"))
+  .def("current",
+      py::overload_cast<const std::size_t, const std::size_t>(
+      &CMFD::current, py::const_),
+      "Returns the current on a CMFD cell boundary\n\n"
+      "Parameters\n"
+      "----------\n"
+      "G: unsigned int\n"
+      "   Int which gives the # of the flux energy group"
+      "surf: unsigned int\n"
+      "   Int which gives the CMFD surface index\n"
```

Comment:
Here is a case where you don't need to say "Int which ...", because the int type is already specified on the line above.